Chummer/data/vehicles.xml

```diff
@@ -10203,6 +10203,236 @@
         </gear>
       </gears>
     </vehicle>
+    <vehicle>
+      <id>be13725c-d5cd-4855-96f6-a07de090b58d</id>
+      <name>Eurocar Escape FüKomKW</name>
+      <source>SOTG</source>
+      <page>16</page>
+      <accel>3</accel>
+      <armor>12</armor>
+      <avail>10</avail>
+      <body>16</body>
+      <category>Trucks</category>
+      <cost>145000</cost>
+      <handling>3/4</handling>
+      <pilot>2</pilot>
+      <sensor>4</sensor>
+      <speed>3</speed>
+      <gears>
+        <gear>
+          <name>Sensor Array</name>
+          <rating>1</rating>
+          <maxrating>6</maxrating>
+        </gear>
+      </gears>
+      <mods>
+        <name rating="2">Anti-Theft System</name>
+        <name>Standard Drone Rack (Small)</name>
+        <name>Off-Road Suspension</name>
+        <name>Amenities (Low)</name>
+        <name>Multifuel Engine</name>
+        <name>Satellite Link</name>
+        <name>Winch (Basic)</name>
+        <name>Searchlight</name>
+        <name>Retrans Unit</name>
+        <name select="Command center with own Host">Special Equipment</name>
+        <name>Enhanced Image Screens</name>
+        <name>Increased Seating</name>
+      </mods>
+      <seats>10</seats>
+    </vehicle>
+    <vehicle>
+      <id>3385c5ce-0224-41be-bede-ef908f0af007</id>
+      <name>Eurocar Escape MzKW D</name>
```

Comment:
"MzKW" I assume should be translated to "Multi-Purpose Vehicle" or something similar.